frame/uniques/src/functions.rs

```diff
@@ -25,6 +25,19 @@ use frame_support::{
 use sp_runtime::{DispatchError, DispatchResult};
 
 impl<T: Config<I>, I: 'static> Pallet<T, I> {
+
+	/// Perform a transfer of an item from one account to another within a collection.
```

Comment:
These function here are developer facing, not directly user facing. I assume you are aware of it since you want to improve developer exp, but i am just re-iterating it.

Reply:
Yep - the goal is to remove all lint errors :)